File: backend/src/test/java/org/luxons/sevenwonders/game/cards/HandRotationDirectionTest.java
package org.luxons.sevenwonders.game.cards;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class HandRotationDirectionTest {

    @Test
    public void testAgesDirections() {
        assertEquals(HandRotationDirection.LEFT, HandRotationDirection.forAge(1));
        assertEquals(HandRotationDirection.RIGHT, HandRotationDirection.forAge(2));
        assertEquals(HandRotationDirection.LEFT, HandRotationDirection.forAge(3));
    }
}

File: game-engine/src/test/kotlin/org/luxons/sevenwonders/game/cards/HandRotationDirectionTest.kt
package org.luxons.sevenwonders.game.cards

import org.junit.Test

import org.junit.Assert.assertEquals

class HandRotationDirectionTest {

    @Test
    fun testAgesDirections() {
        assertEquals(HandRotationDirection.LEFT, HandRotationDirection.forAge(1))
        assertEquals(HandRotationDirection.RIGHT, HandRotationDirection.forAge(2))
        assertEquals(HandRotationDirection.LEFT, HandRotationDirection.forAge(3))
    }
}
